web/styles/zulip.css

```diff
@@ -124,6 +124,8 @@ body {
     --modal-container-subscriptions-display-type-height: 44px;
     /* .modal-footer */
     --modal-container-subscriptions-modal-footer-height: 60px;
+    /* Width of input elements in modals/popovers */
+    --popoover-input-width: 206px;
```

Comment:
`popover` is mispelled, but we should call this `modal-input-width` anyway I think, and make a few variables with different names for the different use cases -- `settings-input-width`, for example, for the usage outside `modals.css`.

Also the `modals.css` change will break the one modal we have in the portico. @sahil839 we should really do something about that -- maybe the best thing would be to move all the CSS variables from `zulip.css` to a new `variables.css` that we can import from multiple places?

Reply:
OK

Reply:
I dropped this commit in that case. Best to do this in a separate PR.